.coveragerc

```diff
@@ -0,0 +1,47 @@
+# .coveragerc for edx-platform
+[run]
+data_file = reports/.coverage
+source =
+    cms
+    common/djangoapps
+    common/lib/calc
+    common/lib/capa
+    common/lib/xmodule
+    lms
+    openedx/core/djangoapps
```

Comment:
We should add pavelib, to encourage continuing to add to coverage there too. It is part of the codebase.